docroot/modules/custom/mass_feedback_loop/src/Form/MassFeedbackLoopPerNodeForm.php

```diff
@@ -32,7 +32,6 @@ class MassFeedbackLoopPerNodeForm extends FormBase {
    */
   public function __construct(MassFeedbackLoopContentFetcher $content_fetcher) {
     $this->contentFetcher = $content_fetcher;
-    $this->request = $this->getRequest();
```

Comment:
Not related to the tag removal, but this was not being used and it was causing deprecation messages on the per-node feedback page.